libs/deck-layers/src/layers/context/ContextLayer.ts

```diff
@@ -87,6 +92,7 @@ export class ContextLayer<PropsT = {}> extends CompositeLayer<_ContextLayerProps
             return [
               new GeoJsonLayer(mvtSublayerProps, {
                 id: `${props.id}-boundaries`,
+                onViewportLoad: this._onViewportLoad,
```

Comment:
as we are now doing anything custom with the callback now why not removing the `_onViewportLoad` method and use `this.props.onViewportLoad` callback directly?